frontend/elements/src/components/icons/styles.sass

```diff
@@ -35,7 +35,7 @@
 .exclamationMark
   @extend .icon
   fill: variables.$error-color
-  padding-right: 5px
+  padding: 1px 6px 0 4px
```

Comment:
Is it necessary for the icon to have left padding? This would create additional space between the padding of the error message box and the icon. Alternatively, you could adjust the padding of the error box to achieve the same effect.

Reply:
You are right, and that’s definitely something I also considered. I decided to try this way in case the icon is replaced in the future or modified in size, etc the spacing could be adjusted just in the icon without influencing the rest of error message box. It is intended the icon to have 8px left so it’s not too close to the border.

Reply:
Ok, I can't verify this one, hence consider it as resolved. But is there a template in figma for the error box?

Reply:
With the applied suggestions, just play around, what looks the best.